tests/config/default.config.ts

```diff
@@ -63,7 +63,7 @@ const config: folio.Config<CommonOptions & PlaywrightOptions> = {
 const browserNames = ['chromium', 'webkit', 'firefox'] as BrowserName[];
 for (const browserName of browserNames) {
   const executablePath = getExecutablePath(browserName);
-  if (executablePath && !process.env.FOLIO_WORKER_INDEX)
+  if (executablePath && !process.env.TEST_WORKER_INDEX)
```

Comment:
Where is the other side of this? Or it already landed in folio?

Reply:
Yep.